lib/Table/TableAutomaticStylesStratum.ts

```diff
@@ -166,11 +160,54 @@ export default class TableAutomaticStylesStratum extends LoadableStratum(
       return true;
   }
 
+  @computed get showDisableTimeOption() {
+    // Return nothing if no row groups or if time column doesn't have at least one interval
+    if (
+      this.catalogItem.activeTableStyle.rowGroups.length === 0 ||
+      !this.catalogItem.activeTableStyle.moreThanOneTimeInterval
+    )
+      return undefined;
+
+    // Return true if at least 50% of rowGroups only have one unique time interval (i.e. they don't change over time)
+    let flat = 0;
+
+    for (
+      let i = 0;
+      i < this.catalogItem.activeTableStyle.rowGroups.length;
+      i++
+    ) {
+      const group = this.catalogItem.activeTableStyle.rowGroups[i];
+      const dates = group[1]
```

Comment:
is there something special about group[1]?

Reply:
I have renamed same vars to make it a bit more clear - `rowGroups[i]: [string, string]`

So i've now made it

```ts
 const [rowGroupId, rowIds] = this.catalogItem.activeTableStyle.rowGroups[
        i
      ];
```